.gitmodules

```diff
@@ -11,3 +11,6 @@
 [submodule "libs/tinyxml2"]
 	path = libs/tinyxml2
 	url = https://github.com/leethomason/tinyxml2.git
+[submodule "grpc/server/proto"]
+	path = grpc/server/proto
+	url = git@github.com:dronecore/DroneCore-Proto.git
```

Comment:
Did you choose the ssh link here on purpose? Do you know if this works for anyone without github ssh key?

Reply:
Good point :-). Gonna change this.